kubernetes/helm/ckbox-development-stack/values.yaml

```diff
@@ -0,0 +1,52 @@
+ckbox:
+  secret:
+    data:
+      DATABASE_HOST: ckbox-mysql
+      DATABASE_USER: root
+      DATABASE_PASSWORD: root-password
```

Comment:
If mysql is creating `ckbox-on-premises` database then we need to use `DATABASE_DATABASE: ckbox-on-premises` for ckbox as well, as the default value is `cs-on-premises`.

Reply:
Added